sc-tools/sc-server/src/sc-server-impl/sc_server.hpp

```diff
@@ -146,6 +147,12 @@ class ScServer
     m_instance->get_elog().write(channel, errorMessage);
   }
 
+  void ClearLogOptions()
+  {
+    delete m_log;
+    m_log = new ScServerLog(m_instance, SC_SERVER_CONSOLE_TYPE, "", SC_SERVER_INFO_LEVEL);
+  }
+
```

Comment:
Are we using this to be able to write to console no matter what settings are defined in the config? We should probably bring the options back after doing this, because otherwise our logs won't be predictable.